doc/connectivity/bluetooth/api/mesh/shell.rst

```diff
@@ -324,47 +342,86 @@ The Proxy Client model is an optional mesh subsystem that can be enabled through
+
+	Set the target Configuration server for the Configuration Client model.
 
-The Configuration Client model is an optional mesh subsystem that can be enabled through the :kconfig:option:`CONFIG_BT_MESH_CFG_CLI` configuration option. If included, the Bluetooth mesh shell module instantiates a Configuration Client model for configuring itself and other nodes in the mesh network.
+	* ``Dst_addr``: The address of the target Configuration Server.
+	* ``NetKeyIndex``: The used network key index.
 
-The Configuration Client uses the general messages parameters set by ``mesh dst`` and ``mesh netidx`` to target specific nodes. When the Bluetooth mesh shell node is provisioned, the Configuration Client model targets itself by default. When another node has been provisioned by the Bluetooth mesh shell, the Configuration Client model targets the new node. The Configuration Client always sends messages using the Device key bound to the destination address, so it will only be able to configure itself and mesh nodes it provisioned.
+``mesh models cfg target get``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-``mesh timeout [timeout in seconds]``
--------------------------------------
+	Get the target Configuration server for the Configuration Client model.
+
+``mesh models cfg help``
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+	Print information for the Configuration Client shell module.
+
+``mesh models cfg reset``
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+	Reset the target device.
+
+``mesh models cfg timeout [timeout in seconds]``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 	Get and set the Config Client model timeout used during message sending.
 
 	* ``timeout in seconds``: If present, set the Config Client model timeout in seconds. If omitted, the current timeout is printed.
 
 
-``mesh get-comp [page]``
-------------------------
+``mesh models cfg get-comp [page]``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 	Read a composition data page. The full composition data page will be printed. If the target does not have the given page, it will return the last page before it.
 
 	* ``page``: The composition data page to request. Defaults to 0 if omitted.
 
 
-``mesh beacon [val: off, on]``
-------------------------------
+``mesh models cfg beacon [val: off, on]``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 	Get or set the network beacon transmission.
 
 	* ``val``: If present, enables or disables sending of the network beacon. If omitted, the current network beacon state is printed.
 
 
-``mesh ttl [ttl: 0x00, 0x02-0x7f]``
------------------------------------
+``mesh models cfg ttl [ttl: 0x00, 0x02-0x7f]``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 	Get or set the default TTL value.
 
 	* ``ttl``: If present, sets the new default TTL value. If omitted, the current default TTL value is printed.
 
 
-``mesh friend [val: off, on]``
-------------------------------
+``mesh models cfg friend [val: off, on]``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
Why did you change `----` by `^^^^`?

Reply:
I added one additional level:  Models->Configuration Client. Before it was just Configuration Client. 
![image](https://user-images.githubusercontent.com/35562897/186902668-0c99b6f0-36bb-4acc-b237-e65108efbc06.png)

Reply:
I didn't dig so deeply into doc rules. Thanks for explanation :)